engine/consensus/matching/core.go

```diff
@@ -415,11 +415,6 @@ func (c *Core) checkSealing() error {
 		Str("next_unsealed_results", nextUnsealeds.String()).
 		Logger()
 
-	if len(sealableResults) == 0 {
-		lg.Info().Msg("checking sealing finished. No results can be sealed yet")
-		return nil
-	}
```

Comment:
The reason to remove is that, if consensus nodes are behind, there isn’t any incoporated results in the mempool, then the requestPendingReceipt will never be triggered, therefore never be able to catch up.